internal/generator/generator.go

```diff
@@ -349,6 +349,11 @@ func NewGenerator(
 		buildValues.PodSecurityContext.OnRootMismatch = true
 	}
 
+	oauth2proxyDomain := helpers.GetEnv("LAGOON_FEATURE_FLAG_OAUTH2PROXY_DOMAIN", generator.EnvironmentVariables, generator.Debug)
+	if oauth2proxyDomain != "" {
+		buildValues.Oauth2ProxyDomain = oauth2proxyDomain
+	}
```

Comment:
Change to use `CheckFeatureFlag` function.

```suggestion
	oauth2proxyDomain := CheckFeatureFlag("OAUTH2PROXY_DOMAIN", buildValues.EnvironmentVariables, generator.Debug)
	if oauth2proxyDomain != "" {
		buildValues.Oauth2ProxyDomain = oauth2proxyDomain
	}
```

Reply:
This may also need some thought to deal with potentially multiple/other things, but for now/testing this is ok